pkgs/top-level/python-aliases.nix

```diff
@@ -474,6 +474,7 @@ mapAliases ({
   pyres = throw "pyres has been removed, since it is abandoned and broken"; # added 2023-06-20
   pygeoip = throw "pygeoip has been removed, since it wash archived in April 2018"; # added 2024-07-27
   pygeos = throw "pygeos has been removed, since it was merged with shapely 2.0"; # added 2024-05-26
+  pyreqwest-impersonate = throw "pyreqwest has been renamed to primp"; # added 2024-10-22
```

Comment:
Looks good to me as it's not a drop-in replacement (`import primp` vs. ìmport pyreqwest_impersonate').